tests/security/ibmtss/ibmtss_basic_function.pm

```diff
@@ -0,0 +1,67 @@
+# Copyright 2021 SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Summary: Update IBM's Trusted Computing Group Software Stack (TSS) to the latest version.
+#          IBM has tested x86_64, s390x and ppc64le, we only need cover aarch64
+#          This test module covers basic function test
+# Maintainer: rfan1 <richard.fan@suse.com>
+# Tags: poo#101088, poo#102792, tc#1769800
+
+use base 'opensusebasetest';
+use base 'consoletest';
+use strict;
+use warnings;
+use testapi;
+use utils 'zypper_call';
+use version_utils 'is_sle';
+use registration 'add_suseconnect_product';
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    # Install required develop packages
+    if (is_sle('>=15')) {
+        add_suseconnect_product('sle-module-desktop-applications');
+        add_suseconnect_product('sle-module-development-tools');
+    }
+    zypper_call('in -t pattern devel_basis');
+    zypper_call('in openssl-devel');
+
+    # Install emulated tpm server
+    zypper_call('in ibmswtpm2');
+
+    # Swith to root console to start tpm server in backaround
+    select_console('root-console');
```

Comment:
why not select root console at the beginning of the test?

Reply:
Thanks, let me fix it